docs/source/kinds_of_types.rst

```diff
@@ -986,6 +986,9 @@ annotated the first example as the following:
    def squares(n: int) -> Generator[int, None, None]:
        for i in range(n):
            yield i * i
+           
+This is slightly different from using ``Iterable[int]`` or ``Iterator[int]``,
+since generators have a ``close()`` method that generic iterables don't.
```

Comment:
Generators have also ``throw`` and ``send``, maybe you could say explicitly that if one wants to call ``send``, ``throw`` or ``close``, then one should use ``Generator[T, None, None]`` (this can actually be a generic type alias). Maybe you can illustrate this with a short example, like:
```python
x: Iterator[int]
x.send(None) # Error!
x.close() # Error!
y: Generator[int, None, None]
... # etc.
```